apps/www/components/Trial/Form.js

```diff
@@ -323,10 +325,9 @@ const Form = (props) => {
                       )}
                     >
                       <ArrowForwardIcon
-                        style={{ cursor: 'pointer' }}
+                        style={{ cursor: 'pointer', color: '#fff' }}
```

Comment:
Change works for dark mode …

<img width="623" alt="Bildschirmfoto 2022-12-19 um 19 16 15" src="https://user-images.githubusercontent.com/331800/208492838-bd2d9818-c5ec-4dd0-a89c-8dafd0f35302.png">

… but not for bright mode anymore.

<img width="622" alt="Bildschirmfoto 2022-12-19 um 19 16 19" src="https://user-images.githubusercontent.com/331800/208492890-e3256b97-bd9d-4194-a444-42d35638570a.png">

Not sure why you nailed that one.

Form seems to be good enough without it? Here an example from Probelesen-Page on prod.

<img width="705" alt="Bildschirmfoto 2022-12-19 um 19 18 26" src="https://user-images.githubusercontent.com/331800/208493157-922a4fd0-9d05-43a5-8c7e-722313149c6d.png">

<img width="735" alt="Bildschirmfoto 2022-12-19 um 19 18 21" src="https://user-images.githubusercontent.com/331800/208493223-64566af9-3792-4f00-b399-3d734ff0b011.png">

Reply:
Ah it was for the case with `background: primary` (black on green doesn't look great)
You are right, in all other cases it doesn't make a lick of sense. Will make it more specific.